roles/container-engine/containerd/templates/config.toml.j2

```diff
@@ -19,6 +19,9 @@ disabled_plugins = ["restart"]
   stream_server_address = "127.0.0.1"
   max_container_log_line_size = {{ containerd_config.max_container_log_line_size }}
   sandbox_image = "{{ pod_infra_image_repo }}:{{ pod_infra_image_tag }}"
+{% if kubelet_cgroup_driver is defined and kubelet_cgroup_driver == "systemd" %}
```

Comment:
There is already a variable for that purpose (`containerd_use_systemd_cgroup`) maybe use it instead

Reply:
Thanks, makes sense. 

We should also fix autodetection for containerd and set cgroup driver accordingly if overridden manually. Probably best to do this in another PR.